src/index.html

```diff
@@ -1,21 +1,12 @@
 <!DOCTYPE html>
-<html>
```

Comment:
You can read about it and much more https://google.github.io/styleguide/htmlcssguide.html